dot-studio/golang

```diff
@@ -29,44 +29,40 @@ DOC
 function setup_go_workspace() {
   # Verify that we are using the Go Scaffolding
   if [[ ! "$pkg_scaffolding" =~ "scaffolding-go" ]]; then
-    echo "ERROR: Unable to setup Go Workspace."
-    echo "${FUNCNAME[0]}: requires the 'pkg_scaffolding' variable to exist in your plan.sh"
+    error "Unable to setup Go Workspace."
+    error "${FUNCNAME[0]}: requires the '$(yellow pkg_scaffolding)' variable to exist in your plan.sh"
     return 2
-  fi
+  fi;
 
   # Verify if the Go Workspace has been already configured
-  if [[ $scaffolding_go_workspace_src  ]]; then
+  if [[ -n $scaffolding_go_workspace_src ]]; then
     return 0
   fi
 
   # Currently the scaffolding-go has all the logic to setup the
   # Go Workspace, lets leverage all its functionality
-  echo "=> Configuring Go Workspace. [please wait]"
-  export scaffolding_go_gopath="${scaffolding_go_gopath:-/hab/cache/src/go}"
+  log_line "${YELLOW}Configuring Go Workspace${NC}"
+
+  export scaffolding_go_gopath="${scaffolding_go_gopath:-/hab/cache/src/scaffolding-go-gopath}"
   install "$pkg_scaffolding" >/dev/null
-  source "$(hab pkg path "$pkg_scaffolding")/lib/"*
-  export PATH="$PATH:${scaffolding_go_gopath}/bin"
-
-  # Verify if we have the go_before callback
-  # If not lets setup the workspace ourself
-  go_before_callback=scaffolding_go_before
-  if [[ "$(type -t $go_before_callback)" = function ]]; then
-    eval "$go_before_callback"
-  else
-    mkdir -p "${scaffolding_go_workspace_src}/${scaffolding_go_base_path}" || return
-    if [[ ! -e "$scaffolding_go_pkg_path" ]]; then
-      ln -s /src "$scaffolding_go_pkg_path"
-    fi
-  fi
+
+  source "$(pkg_path_for "$pkg_scaffolding")/lib/scaffolding.sh"
```

Comment:
Is the `lib/scaffolding.sh` file the only one, because before it was sourcing all the file in the lib directory.

Reply:
Yes, scaffoldings only have that file. https://www.habitat.sh/docs/glossary/#scaffolding 
>lib/scaffolding.sh File
To create scaffolding, a package must contain a lib/scaffolding.sh file which gets sourced by the build program running Bash.

The **"Wrapper Scaffolding"** is the one that has the `lib/scaffolding_overrides.sh` but we are doing the magic inside the `scaffolding_load` function. 🥇